rclcpp/include/rclcpp/executor.hpp

```diff
@@ -347,6 +347,7 @@ class Executor
 
   /// Spin (blocking) until the future is complete, it times out waiting, or rclcpp is interrupted.
   /**
+   * Handles one work item after the other, hence, it does not support parallel processing.
```

Comment:
I would prefer to add `can do` instead of `cannot do` here. how about the following?

```suggestion
   * The implementation is suitable for a single-thread model of execution.
```